src/community/jdbcstore/src/main/resources/jdbcstore.properties

```diff
@@ -44,4 +44,4 @@ pool.validationQuery=SELECT now()
 deleteDestinationOnRename=true
 
 # leave the following directories on the filesystem -- no spaces!
-ignoreDirs=data,jdbcstore,workspaces,layergroups,temp,tmp
+ignoreDirs=data,jdbcstore,workspaces,layergroups,temp,tmp,logs
```

Comment:
@jodygarnett I know that is convention; that is why this is a configuration setting, these are merely the default values.
I thought that whatever is in workspaces and layergroups managed by the catalog (jdbcconfig), therefore they need not be imported in the database as well. Don't know how to avoid them being imported if the dirs are mixed

@tbarsballe the jdbcconfig files can theoretically be managed by jdbcresource